tutorial/markdown/java/spring-data/spring-data.md

```diff
@@ -0,0 +1,427 @@
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- Code Editor or an Integrated Development Environment (e.g., [Eclipse](https://www.eclipse.org/ide/))
+- [Java SDK v1.8 or higher installed](https://www.oracle.com/java/technologies/ee8-install-guide.html)
+- [Gradle Build Tool](https://gradle.org/install/)
+
+### Source Code
+
+The sample source code used in this tutorial is [published on GitHub](https://github.com/couchbase-examples/java-springboot-quickstart).
+To obtain it, clone the git repository with your IDE or execute the following command:
+
+```shell
+git clone https://github.com/couchbase-examples/java-springdata-quickstart.git
+```
+
+### Install Dependencies
+
+Gradle dependencies:
+
+```groovy
+implementation 'org.springframework.boot:spring-boot-starter-web'
+// spring data couchbase connector
+implementation 'org.springframework.boot:spring-boot-starter-data-couchbase'
+// swagger ui
+implementation 'org.springdoc:springdoc-openapi-ui:1.6.6'
+```
+
+Maven dependencies:
+
+```xml
+<dependency>
+  <groupId>org.springframework.boot</groupId>
+  <artifactId>spring-boot-starter-data-couchbase</artifactId>
+</dependency>
+<dependency>
+  <groupId>org.springframework.boot</groupId>
+  <artifactId>spring-boot-starter-web</artifactId>
+</dependency>
+<dependency>
+  <groupId>org.springdoc</groupId>
+  <artifactId>springdoc-openapi-ui</artifactId>
+  <version>1.6.6</version>
+</dependency>
+```
+
+#### What We'll Cover
+
+- [Cluster Connection Configuration](#cluster-connection-configuration) – Configuring Spring Data to connect to a Couchbase cluster.
+- [Database Initialization](#database-initialization) – Creating required database structures upon application startup
+- [CRUD operations](#create-or-update-a-profile) – Standard create, update and delete operations.
+- [Custom SQL++ queries](#search-profiles-by-text) – Using [SQl++](https://www.couchbase.com/sqlplusplus) with Spring Data.
+
+#### Useful Links
+
+- [Spring Data Couchbase - Reference Documentation](https://docs.spring.io/spring-data/couchbase/docs/current/reference/html/)
+- [Spring Data Couchbase - JavaDoc](https://docs.spring.io/spring-data/couchbase/docs/current/api/)
+
+### Database Server Configuration
+
+Spring Data couchbase connector can be configured by providing a `@Configuration` [bean](https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#beans-definition) that extends [`AbstractCouchbaseConfiguration`](https://docs.spring.io/spring-data/couchbase/docs/current/api/org/springframework/data/couchbase/config/AbstractCouchbaseConfiguration.html).
+The sample application provides a configuration bean that uses default couchbase login and password:
+
+```java
+@Configuration
+public class CouchbaseConfiguration extends AbstractCouchbaseConfiguration {
+
+  @Override
+  public String getConnectionString() {
+    // capella
+    // return "couchbases://cb.jnym5s9gv4ealbe.cloud.couchbase.com";
```

Comment:
Please remove the database identifier from here. You can use cb.xyz.cloud.couchbase.com

Reply:
changed to cb.xyz.cloud.couchbase.com